### ISSUE TITLE
Home Page: tab order navigation is not reinforced by visual cues on page element.

### ISSUE THREAD

**Issue by MedicalTest** [codalab/codalab-competitions#196]
Issue:  Home Page: tab order navigation is not reinforced by visual cues on page element.

![tab](https://f.cloud.github.com/assets/5091775/1134986/d41c5fb2-1c01-11e3-8838-b037b0c6fe08.png)

Steps:
     1.  Launch CodaLab  (http://cl-test1-ubuntu.cloudapp.net/help/about.html)
     2.  Click on HomePage 
     3.  Begin tabbing…

Actual result:  visual cue on menu items to support tab order is missing.  (see attached, “tab.png”)

Expected result:  visual cue on menu items to support tab order.

Configuration details
Build date: 9/11/2013, ~2 p.m.
Build: e1c2a80d6ac48b37d9c39a493db3d0a397f40e00 
OS: Win7
Browser(s):   all supported
